.github/workflows/auto_changelog.yml

```diff
@@ -21,3 +21,4 @@ jobs:
         script: |
           const { processAutoChangelog } = await import('${{ github.workspace }}/tools/pull_request_hooks/autoChangelog.js')
           await processAutoChangelog({ github, context })
+        github-token: ${{ secrets.COMFY_ORANGE_PAT || secrets.GITHUB_TOKEN }}
```

Comment:
Just to be sure, this being after `script:` doesn't matter, right?

Reply:
Nah